common-lib/src/cmr/common/util.clj

```diff
@@ -1022,6 +1022,32 @@
           false))
       false)))
 
+;; Note: Similar code exists at gov.nasa.echo.kernel.service.authentication
+(def URS_TOKEN_MAX_LENGTH 100)
+(def BEARER "Bearer ")
+
+;; TODO - remove legacy token check after legacy token retirement
+(defn is-legacy-token?
+  "There are two uses cases captured by this test, the Legacy token and the
+   new style legacy token made to behave like a legacy token but are prefixed
+   with EDL- to aid in indentification. This function will not match very short
+   JWT tokens.
+   Note: Similar code exists at gov.nasa.echo.kernel.service.authentication."
+  [token]
+  (or (<= (count token) URS_TOKEN_MAX_LENGTH)
+      (string/starts-with? token "EDL-")
+      (string/starts-with? token (str BEARER "EDL-"))))
```

Comment:
I think this is dead code as well: (string/starts-with? token (str BEARER "EDL-")))).  Please check with Jay first though.